### PR TITLE
Major refactoring of main

### PR DESCRIPTION
**Pull request by BlackSound1**
- Main2.cpp has been majorly reorganized. Functions moved to the bottom, globals moved to the top, more comments, unused things removed, etc.
- Removed useless tattoo texture
- Minor refactoring of CrowdObjects files. More on the way

I stopped here because it's already pretty major. Refactoring other files is less important for now.

### CODE REVIEW

File: src/main2.cpp [hunk 1]
```diff
@@ -176,123 +122,38 @@ TexturedNormalledVertex texturedCubeVertexArray[] = {
...
+bool scoreIncremented = false;
 int renderAs = GL_TRIANGLES;
 int shaderProgram;
 double lastMousePosX, lastMousePosY, lastMousePosZ;
 float FOV = 70;
 float AR = (float) WIDTH / (float) HEIGHT;
 float near = .01;
 float far = 50;
-float translateW = 0, translateY = 0, translateZ = 0;
+float translateW = 0;
```
Comment (MillerSty) [inline src/main2.cpp:141]: Im not sure these translations are used anymore

Comment (MillerSty) [reply src/main2.cpp:141]: (translateX,translateY,translateZ)

File: src/main2.cpp [hunk 2]
```diff
@@ -459,19 +319,17 @@ int main(int argc, char* argv[])
...
 	Material woodMaterial(.5f, .60f, .5f, .002f, woodTextureID, shaderProgram); //this is you matt
-	Material tattooMaterial(0.5f, 0.2f, 0.5f, 0.002f, tattooTextureID, shaderProgram);
-	Material skinMaterial(.1f, .0f, .66f, .001f, plasticTextureID, shaderProgram); //this is skin
-	Material skyMaterial(.3f, .001f, .9f, .0001f, plasticTextureID, shaderProgram); //Flat blue sky
-	Material bleacherMaterial(.6f, .90f, .6f, .12f, aluminumTextureID, shaderProgram); //Flat blue sky
-
-	//need new way to do bleachers
+	Material skinMaterial(.1f, .0f, .66f, .001f, plasticTextureID, shaderProgram); // This is skin
+	Material skyMaterial(.3f, .001f, .9f, .0001f, plasticTextureID, shaderProgram); // Flat blue sky
+	Material bleacherMaterial(.6f, .90f, .6f, .12f, aluminumTextureID, shaderProgram); // Metal
```
Comment (MillerSty) [inline src/main2.cpp:324]: we're actually just using metalMaterial for bleachers so this could probably be removed

File: src/main2.cpp [hunk 3]
```diff
@@ -793,54 +634,60 @@ int main(int argc, char* argv[])
...
 
+            // Draw court and skybox
             SceneObj.sphereVao = unitSphereAO;
             SceneObj.sphereVertCount = (int) vertexIndicesSphere.size();
             SceneObj.SetAttr(rotationMatrixW, renderAs, shaderProgram);
             SceneObj.SetVAO(unitCubeAO, gridAO);
             SceneObj.DrawScene(false);  // Draw scene without the skybox, so it can't be used to make shadows on the scene
 
-			//This is all for bleacher			
+			// This is all for the bleachers
```
Comment (MillerSty) [inline src/main2.cpp:670]: ideally bleachers and red court could get combined into scene objects.

**Review by MillerSty — APPROVED** [review]
Looks good though, comments are just suggestions for future cleanup

File: src/main2.cpp (hunk 2 above)
Comment (MillerSty) [reply src/main2.cpp:324]: Note: could make bleacherMaterial use the metalTextureID and turn the spec down, or shininess down?

File: src/main2.cpp (hunk 1 above)
Comment (BlackSound1) [reply src/main2.cpp:141]: I'll definitely look into it

File: src/main2.cpp (hunk 2 above)
Comment (BlackSound1) [reply src/main2.cpp:324]: Sure, will do

File: src/main2.cpp (hunk 3 above)
Comment (BlackSound1) [reply src/main2.cpp:670]: Perhaps. I can try that

**Pull request merged**